tests/nnstreamer_filter_openvino/unittest_openvino.cc

```diff
@@ -511,6 +582,565 @@ TEST (tensor_filter_openvino, open_and_close_2_n)
+      "models", str_test_model.assign (MODEL_BASE_NAME_MOBINET_V2)
+      .append (TensorFilterOpenvino::extBin).c_str (),
+      NULL);
+
+  {
+    TensorFilterOpenvinoTest tfOvTest (str_test_model.assign (test_model_xml),
+        str_test_model.assign (test_model_bin));
+    // A test case when the number of ranks of a tensor in the input exceed is exceeded NNS_TENSOR_RANK_LIMIT
+    std::string name_output = std::string ("output");
+    InferenceEngine::SizeVector dims;
+    InferenceEngine::OutputsDataMap outDataMap;
+    InferenceEngine::Data *data;
+
+    dims = InferenceEngine::SizeVector (NNS_TENSOR_RANK_LIMIT + 1, 1);
+    data = new InferenceEngine::Data (name_output, dims,
+        InferenceEngine::Precision::FP32, InferenceEngine::ANY);
+    outDataMap[name_output] = InferenceEngine::DataPtr (data);
+
+    tfOvTest.setOutputsDataMap (outDataMap);
+    ret = tfOvTest.loadModel (ACCL_CPU);
+    private_data = (gpointer) &tfOvTest;
+
+#ifdef __OPENVINO_CPU_EXT__
+    EXPECT_EQ (ret, 0);
+#else
+    EXPECT_NE (ret, 0);
+    EXPECT_EQ (ret, TensorFilterOpenvino::RetENoDev);
+#endif
+
+    /* prepare properties */
+    prop = g_new0 (GstTensorFilterProperties, 1);
+    ASSERT_TRUE (prop != NULL);
+    prop->fwname = fw_name;
+
+    /* Test getOutputDimension () */
+    ASSERT_TRUE (fw->getOutputDimension);
+    ret = fw->getOutputDimension (prop, &private_data, &nns_tensors_info);
+    EXPECT_NE (ret, 0);
+    g_free (prop);
+  }
+
+  g_free (test_model_xml);
+  g_free (test_model_bin);
+}
+
+/**
+ * @brief A test case for the helper function, convertFromIETypeStr ()
+ */
+TEST (tensor_filter_openvino, convertFromIETypeStr_0)
+{
+  const gchar *root_path = g_getenv ("NNSTREAMER_BUILD_ROOT_PATH");
+  const std::vector<std::string> ie_suport_type_strs = {
+    "I8",
+    "I16",
+    "I32",
+    "U8",
+    "U16",
+    "FP32",
+  };
+  const std::vector<tensor_type> nns_support_types = {
+    _NNS_INT8,
+    _NNS_INT16,
+    _NNS_INT32,
+    _NNS_UINT8,
+    _NNS_UINT16,
+    _NNS_FLOAT32,
+  };
+  std::string str_test_model;
+  gchar *test_model_xml;
+  gchar *test_model_bin;
+
+  /* supposed to run test in build directory */
+  if (root_path == NULL)
+    root_path = "..";
+
+  test_model_xml = g_build_filename (root_path, "tests", "test_models",
+      "models", str_test_model.assign (MODEL_BASE_NAME_MOBINET_V2)
+      .append (TensorFilterOpenvino::extXml).c_str (),
+      NULL);
+  test_model_bin = g_build_filename (root_path, "tests", "test_models",
+      "models", str_test_model.assign (MODEL_BASE_NAME_MOBINET_V2)
+      .append (TensorFilterOpenvino::extBin).c_str (),
+      NULL);
+
+  {
+    TensorFilterOpenvinoTest tfOvTest (str_test_model.assign (test_model_xml),
+        str_test_model.assign (test_model_bin));
+    for (size_t i = 0; i < ie_suport_type_strs.size (); ++i) {
+      tensor_type ret_type;
+
+      ret_type =tfOvTest.convertFromIETypeStr (ie_suport_type_strs [i]);
```

Comment:
Minor comment: need a space between `=` and `tfOvTest`. And, there are a few lines with the same problem.

Reply:
@dongju-chae Fixed. Thnx.